composer.json

```
@@ -2,28 +2,19 @@
   "name": "forkawesome/fork-awesome",
   "description": "A fork of the iconic font and CSS framework",
   "keywords": ["font", "awesome", "fontawesome", "forkawesome", "icon",  "bootstrap"],
```

I am wondering whether we should also update the keywords for the project :thinking: 

```suggestion
  "keywords": ["font", "awesome", "fontawesome", "forkawesome", "icon", "icon-font", "font-awesome", "fork-awesome"],
```

We do not have any relation with https://github.com/twbs/bootstrap other than the fact that we use it as a framework ( library ) to generate the documentation website. That does not warrant being a keyword for the project ( if that were the case, we might as well add https://github.com/jquery/jquery ( which is a direct dependency for the version of bootstrap we use ), javascript, etc as keywords for the project too )